docs/tutorials/restless_measurements.rst

```diff
@@ -0,0 +1,197 @@
+        FixedFrequencyTransmon,
+    )
+    from qiskit_experiments.data_processing.data_processor import DataProcessor
+    from qiskit.test.mock import FakeBogota
+
+    # replace this lines with an IBM Quantum backend to run the experiment.
+    backend = FakeBogota()
+    cals = Calibrations.from_backend(backend, library=FixedFrequencyTransmon())
+
+    # Define the experiment
+    qubit = 2
+    cal_drag = RoughDragCal(qubit, cals, schedule_name='sx', backend=backend)
+
+    # Enable restless measurements by setting the run options and data processor
+    cal_drag.enable_restless(rep_delay=1e-6)
+
+After calling ``enable_restless`` the experiment is ready to be run in a restless
+mode. With a hardware backend this would be done by calling the ``run`` method
+
+.. code:: python
+
+    drag_data_restless = cal_drag.run()
+
+As shown by the example, the code is identical to running a normal
+experiment aside from a call to the method ``enable_restless``. This method
+will set the data processor that post-processes the restless measured shots
+according to the order in which they were acquired. You can also choose
+to keep the standard data processor by providing it to the analysis
+options and telling ``enable_restless`` not to override the data processor.
+
+.. jupyter-execute::
+
+    from qiskit_experiments.data_processing import (
+        DataProcessor,
+        Probability,
+    )
+
+    # define a standard data processor.
+    standard_processor = DataProcessor("counts", [Probability("1")])
+
+    cal_drag = RoughDragCal(qubit, cals, schedule_name='sx', backend=backend)
+    cal_drag.set_analysis_options(data_processor=standard_processor)
+
+    # enable restless mode and set override_processor_by_restless to False.
+    cal_drag.enable_restless(rep_delay=1e-6, override_processor_by_restless=False)
+
+If you run the experiment in this setting you will see that the data is often
+unusable which illustrates the importance of the data processing. As detailed
+in Ref. [2] restless measurements can be done with a wide variety
+of experiments such as fine amplitude and drag error amplifying gate sequences
+as well as randomized benchmarking.
+
+Calculating restless quantum processor speed-ups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can compare the time spent by the quantum processor executing restless and
+standard jobs. This allows us to compute the effective speed-up we gain when
+performing restless experiments. Note that we do not consider any classical
+run-time contributions such as runtime-compilation or data transfer times [3].
+
+The time to run :math:`K` circuits and gather :math:`N` shots for each
+circuit is
+
+.. math::
+
+    \tau^{(x)} = NK\left(\tau^{(x)}_\text{reset}+\tau^{(x)}_\text{delay}+
+    \langle{\tau}_\text{circ}\rangle+\tau_\text{meas}\right),
+
+where :math:`\tau^{(x)}_\text{reset}` and :math:`\tau^{(x)}_\text{delay}`
+are the reset and delay times, respectively. The superscript :math:`(x)`

```

Comment:
Can you clarify the "delay times"? This is bit vague. Is this IO latency or cavity reset? Note that T1 experiment has _delay_ which is intentional.

Reply:
See here: [a448c71](https://github.com/Qiskit/qiskit-experiments/pull/741/commits/a448c71a4e696db4af036c47ee919d088a3b5436) I specified `post measurement delay times` which should be sufficient. I also added the ref to the paper in [667b65d](https://github.com/Qiskit/qiskit-experiments/pull/741/commits/667b65d957bf0802ff1f9a2408cea31b2801977d) so it should be clear.